### PR TITLE
added missing docker images

### PR DESCRIPTION
**Pull request by MLainer1**

### CODE REVIEW

**Review by GuyAfik — APPROVED** [review]
💯 👑 🥇

File: docker_images_metadata.json [hunk 1]
```diff
@@ -3926,6 +3947,368 @@
...
+                "python_version": "3.9.6"
+            },
+            "1.0.0.73692": {
+                "python_version": "3.10.13"
+            },
+            "1.0.0.18446": {
+                "python_version": "3.9.2"
+            }
+        },
+        "l": {
```
Comment (GuyAfik) [inline docker_images_metadata.json]: docker names are not correct O.o

Comment (MLainer1) [reply docker_images_metadata.json]: @ilaner

Comment (MLainer1) [reply docker_images_metadata.json]: fixed

**Pull request merged**